process/coordinator/transactionType_test.go

```diff
@@ -154,6 +154,26 @@ func TestTxTypeHandler_ComputeTransactionTypeScDeployment(t *testing.T) {
 	assert.Equal(t, process.SCDeployment, txType)
 }
 
+func TestTxTypeHandler_ComputeTransactionTypeRecv0AddressWrongTransaction(t *testing.T) {
```

Comment:
Recv0Address ? Is the test name ok?

Reply:
it is receiver 0 address. the special address for deployment.

Reply:
ok